prism.el

```diff
@@ -901,34 +905,40 @@ arguments to set the same faces."
              (symbol-function 'ignore)))
     ;; Avoid saving the file for each variable, which is very slow.
     ;; Save it once at the end.
-    (dolist (var (list 'prism-desaturations 'prism-lightens 'prism-num-faces
-                       'prism-comments-fn 'prism-strings-fn))
+    (dolist (var (list 'prism-desaturations 'prism-lightens 'prism-opacities
+                       'prism-num-faces
+                       'prism-comments-fn 'prism-strings-fn 'prism-parens-fn))
       (customize-save-variable var (symbol-value var))))
   (customize-save-variable 'prism-colors prism-colors))
 
-(cl-defun prism-modify-colors (&key num colors desaturations lightens &allow-other-keys)
-  "Return list of NUM colors modified according to DESATURATIONS and LIGHTENS."
-  (cl-flet ((modify-color (color desaturate lighten)
+(cl-defun prism-modify-colors (&key num colors desaturations lightens opacities &allow-other-keys)
+  "Return list of NUM COLORS modified according to DESATURATIONS, LIGHTENS and OPACITIES."
+  (cl-flet ((modify-color (color desaturate lighten opacity)
                           (--> color
-                            (if (> desaturate 0)
-                                (color-desaturate-name it desaturate)
-                              it)
-                            (if (> lighten 0)
-                                (color-lighten-name it lighten)
-                              it)
-                            ;; FIXME: It seems that these two functions called in sequence
-                            ;; always modify the color, e.g. #ff2afc becomes #fe29fb.
-                            (color-name-to-rgb it)
-                            (-let (((r g b) it))
-                              (color-rgb-to-hex r g b 2)))))
+                               (if (> desaturate 0)
+                                   (color-desaturate-name it desaturate)
+                                 it)
+                               (if (> lighten 0)
+                                   (color-lighten-name it lighten)
+                                 it)
+                               (if (> opacity 0)
+                                   (prism-opacity it opacity))
+                               ;; FIXME: It seems that these two functions called in sequence
+                               ;; always modify the color, e.g. #ff2afc becomes #fe29fb.
+                               (color-name-to-rgb it)
+                               (-let (((r g b) it))
+                                 (color-rgb-to-hex r g b 2)))))
```

Comment:
There seem to be no changes here, so it must be whitespace.  Please remove any whitespace-only changes.